backend/src/api/notification/notifications.api.ts

```diff
@@ -0,0 +1,32 @@
+import { Router } from 'express';
+import { ApiPath, HttpCode, NotificationsApiPath, HttpMethod } from '~/common/enums/enums';
+import { userNotification as userNotificationService } from '~/services/services';
+import { handleAsyncApi } from '~/helpers/helpers';
+import {
+  checkAuth as checkAuthMiddleware,
+} from '~/middlewares/middlewares';
+
+type Args = {
+  apiRouter: Router;
+  userNotificationService: typeof userNotificationService;
+};
+
+const initNotificationApi = ({ apiRouter, userNotificationService }: Args): Router => {
+  const notificationRouter = Router();
+
+  apiRouter.use(ApiPath.NOTIFICATIONS, notificationRouter);
+
+  notificationRouter.get(
+    NotificationsApiPath.$ID,
```

Comment:
```suggestion
    NotificationsApiPath.USER,
```
take user from context